docs/source/onnx_pt2.md

```diff
@@ -1,4 +1,4 @@
-# TorchDynamo-based ONNX Exporter
+# PyTorch2-based ONNX Exporter
```

Comment:
I think using Dynamo is okay? Torchscript was the main concern

Reply:
The issue is that under the hood, ONNX exporter is not really using dynamo, instead, we use `torch.export.export(..., strict=False)` as our upstream. The documentation was wrong (even `dynamo` is used in `torch.onnx.export` API) due to legacy reasons. It could be bc and huge impact if we rename `dynamo` in our API, so we decided to keep it, since it's going to be deleted anyway (as torchscript EOL). 

But in the docs, at least we want to keep the correct information/knowledge to the users what's really going on behind the API.